slasher/Cargo.toml

```diff
@@ -13,8 +13,9 @@ flate2 = { version = "1.0.14", features = ["zlib"], default-features = false }
 lazy_static = "1.4.0"
 lighthouse_metrics = { path = "../common/lighthouse_metrics" }
 filesystem = { path = "../common/filesystem" }
-lmdb = "0.8"
-lmdb-sys = "0.8"
+# mdbx = { git = "https://github.com/vorot93/mdbx-rs", rev = "1150cafb38d6136c863c4618a4eff6a13723537d" }
+mdbx = { git = "https://github.com/michaelsproul/mdbx-rs", branch = "windows" }
```

Comment:
Will switch this over to upstream once https://github.com/vorot93/mdbx-rs/pull/6 is merged (could be soon)

Reply:
Done